### PR TITLE
INFO REQUESTED - CFD-444 - Errors on people page

### PR DESCRIPTION
**Pull request by soniCaH**
Temporary workaround for empty facets on /people and php index errors. Solid solution needs to be picked and implemented. See https://issuetracker.amplexor.com/jira/browse/CFD-444.

### CODE REVIEW

**Comment by soniCaH** [comment]
No errors:

screen shot 2015-04-27 at 12 25 35

No empty facets:

screen shot 2015-04-27 at 12 25 40

*NOTE*: TEMPORARY WORKAROUND

File: capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.module [hunk 1]
```diff
@@ -124,11 +124,13 @@ function _c4m_user_profile_facet_country_alter(&$build, &$settings) {
...
-      ),
-    );
-    $build[$key]["#markup"] = t('@country', $options);
+    if (!empty($allowed_values[drupal_strtoupper($key)])) {
+      $options = array(
+        '@country' => ucwords(
+          drupal_strtolower($allowed_values[drupal_strtoupper($key)])
+        ),
+      );
+      $build[$key]["#markup"] = t('@country', $options);
```
Comment (amitaibu) [inline capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.module]: we could probably use ``format_string()`` instead of ``t()``

Comment (soniCaH) [inline capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.module]: Altered code and pushed again

**Comment by zero2one** [comment]
@soniCaH  needs a re-roll to fix the broken build

**Comment by soniCaH** [comment]
Re-rolling

**Comment by soniCaH** [comment]
Should/will be fixed on capacity4dev (wrong data --> mapping)

**Pull request closed without merging**